mifosng-android/src/main/java/com/mifos/mifosxdroid/online/SurveyQuestionActivity.java

```diff
@@ -7,15 +7,18 @@
 
 import android.content.Intent;
 import android.os.Bundle;
+
 import androidx.fragment.app.Fragment;
 import androidx.viewpager.widget.PagerAdapter;
 import androidx.viewpager.widget.ViewPager;
 import androidx.viewpager.widget.ViewPager.OnPageChangeListener;
 import androidx.appcompat.widget.Toolbar;
+
```

Comment:
Same here, remove extra line

Reply:
Travis CI Build is passed with no conflict

Reply:
The Travis CI will also pass after removing those extra lines. As this project has a big codebase, every line of code matters.